src/common/hooks/use-analytics.ts

```diff
@@ -6,7 +6,7 @@ import { analytics } from '@common/segment-init';
 import { SegmentEvent } from '@segment/analytics-next';
 import { logger } from '@common/logger';
 
-const IGNORED_EVENT_REGEXPS = [/^View \/$/];
+const IGNORED_EVENT_REGEXPS = [/^view \/$/];
```

Comment:
why do we ignore these 🤔

Reply:
It does not add any value I think and pollute the analytics, but we can add it back and see later.